packages/framework-core/src/services/event-store.ts

```diff
@@ -73,6 +73,18 @@ export class EventStore {
     }
   }
 
+  @Trace(TraceActionTypes.CUSTOM)
+  public async storeDispatchedEvent(eventEnvelope: EventEnvelope): Promise<boolean | undefined> {
+    const logger = getLogger(this.config, 'EventStore#storeDispatchedEvent')
+    try {
+      logger.debug('Storing event in the dispatched event store:', eventEnvelope)
+      return await this.config.provider.events.storeDispatched(eventEnvelope, this.config)
+    } catch (e) {
+      logger.debug('Could not store dispatched event. Continue its processing.', eventEnvelope)
```

Comment:
I missed this before: Please log the error too, if not we will swallow it and it would be really hard to find out in the deployed environments. Something like: 
```suggestion
      logger.debug('Could not store dispatched event. Continue its processing.', {error: e, eventEnvelope} )
```

Reply:
Changed in commit 1eb76723145dabcaf935257523f9743dc66f23e7